### DISCUSSION TITLE
Sending a keystroke to an application on bunch close

### DISCUSSION THREAD

**Discussion by moffat** [ttscoff/bunch-docs#337] [Q&A]
Hi,

One of my bunches opens a particular document on google doc. I'd like to close that document when the bunch closes, but not close the browser (Arc). 

Ideally, I'd like to send a {@-w} command to Arc on the bunch's closing, but this doesn't seem to work. 

Any ideas?

Thanks.

**Comment by ttscoff** [comment]
I don't know what ~@w is supposed to do in Arc (I'm not a regular user), but you can send key commands on close with syntax like:

```
!! Arc
- {@w}
```

I looked through the menu and couldn't find ~@w, but I was able to send @w on close without issue. Perhaps you need a different keystroke?

**Comment by moffat** [comment]
Hi @ttscoff,

Yeah, that was a typo, I meant {@w}. And thanks, your solution works!
